### DISCUSSION TITLE
ch12/ch12-01

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#223] [General]
# ch12/ch12-01

<The Go Programming Language>中文版

https://golang-china.github.io/gopl-zh/ch12/ch12-01.html

**Comment by Rowen-Ye** [comment]
这个Switch的用法之前讲过吗。。。。我怎么没见到类型断言在switch

**Reply by sxinh** [reply]
7.13 有示例的
